.esh/commands/reshape_i18n_keys.py

```
@@ -41,7 +41,7 @@
 AFFIX_RE = re.compile(r"^(?:Suffix|Prefix)|(?:Suffix|Prefix)$")
 NUMBERED_RE = re.compile(r"^[A-Za-z][A-Za-z]*\d+$")
 SCREAMING_SNAKE_RE = re.compile(r"^[A-Z][A-Z0-9_]*$")
-UPDATE_LOCALE_DATA_HINT = "搬键请用 `.esh/commands/update_locale_data.py`（get → set(new) → set(old, None)），勿手改各语言 JSON。详见 src/public/locales/locale-edits.md。"
+UPDATE_LOCALE_DATA_HINT = "搬键请用 `.esh/commands/update_locale_data.py`（get → set(new) → set(old, None)），勿手改各语言 JSON。详见 src/public/locales/docs/locale-edits.md。"
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)